src/Middleware/HttpLogging/src/W3CLoggerOptions.cs

```diff
@@ -106,6 +106,11 @@ public TimeSpan FlushInterval
         }
     }
 
+    /// <summary>
```

Comment:
Add a warning about potential security concerns here, similar to the comment on Cookie in `W3CLoggingFields`: https://github.com/dotnet/aspnetcore/blob/bf50999cd8e60d4c683b772dffa9898794a8433a/src/Middleware/HttpLogging/src/W3CLoggingFields.cs#L112-L120

Reply:
CC @blowdart

Reply:
added